ensenso_camera2/launch/stereo_node.launch.py

```diff
@@ -16,6 +16,9 @@ def launch_setup(context, *args, **kwargs):
     link_frame = ensenso.fix_empty_string(LaunchConfiguration("link_frame").perform(context))
     robot_frame = ensenso.fix_empty_string(LaunchConfiguration("robot_frame").perform(context))
     wrist_frame = ensenso.fix_empty_string(LaunchConfiguration("wrist_frame").perform(context))
+    objects_file = ensenso.fix_empty_string(LaunchConfiguration("objects_file").perform(context))
+    objects_frame = ensenso.fix_empty_string(LaunchConfiguration("objects_frame").perform(context))
+    visualization_marker_topic = ensenso.fix_empty_string(LaunchConfiguration("visualization_marker_topic").perform(context))
```

Comment:
```
    visualization_marker_topic = ensenso.fix_empty_string(
        LaunchConfiguration("visualization_marker_topic").perform(context)
    )
```

Reply:
Can I just suggest and merge it here...?
```suggestion
    visualization_marker_topic = ensenso.fix_empty_string(
        LaunchConfiguration("visualization_marker_topic").perform(context)
    )
```

Reply:
Nevermind, that was a test. I will just amend the commit again.